### PR TITLE
Fix tile text blurring on hover

### PR DESCRIPTION
**Pull request by mayank99**
Applying `backface-visibility: hidden` fixes tile's text blurring issue (which has long plagued us on Windows). It makes the "back" of the elements invisible. It helps in our case because we're applying a 3D transform on hover (`translateZ` in thumbnail), which does not work well with subpixels I guess.

I applied the rule to `.iui-tile` to cover all elements (i.e. badge text inside thumbnail) instead of just `.iui-content`.

### CODE REVIEW

**Comment by mayank99** [comment]
Here's a really zoomed in gif showing the issue showing three states:
- no extra rules (i.e. before this PR)
- [`backface-visibility: hidden`](https://developer.mozilla.org/en-US/docs/Web/CSS/backface-visibility) which fixes it.
- [`will-change: transform` ](https://developer.mozilla.org/en-US/docs/Web/CSS/will-change) which also fixes it, but is computationally expensive and should be considered a "last resort" so I left it out of this PR.

Animation

@FlyersPh9 was not able to reproduce the blurring on macOS so this gif should help.
@veekeys @bentleyvk Can you try toggling these rules on your machine to see if the behavior matches the gif?

**Review by FlyersPh9 — APPROVED** [review]

File: src/tile/tile.scss [hunk 1]
```diff
@@ -11,6 +11,7 @@
   flex-direction: column;
   width: $iui-3xl * 3;
   box-shadow: $iui-elevation-2;
+  backface-visibility: hidden;
```
Comment (bentleyvk) [inline src/tile/tile.scss:14]: For me, when this rule is set, text is always a bit blurry:
PR_comments2

Comment (veekeys) [reply src/tile/tile.scss:14]: Same, but it does help on hover...

Comment (mayank99) [reply src/tile/tile.scss:14]: Did you try `will-change: transform` instead?

Comment (mayank99) [reply src/tile/tile.scss:14]: Some other solutions include:
- `transform: translateZ(0)`
- `transform: perspective(1px)`
- `filter: blur(0)`

All from https://stackoverflow.com/questions/14677490/blurry-text-after-using-css-transform-scale-in-chrome

Comment (veekeys) [reply src/tile/tile.scss:14]: will-change does the same, at least at my end..

Comment (bentleyvk) [reply src/tile/tile.scss:14]: Same for me with other rules - text is always blurry. I think I have tried all of these on tooltip without luck.

**Pull request merged**